docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,76 @@
    "id": "840ea662",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    check = SimpleModelComparison(simple_model_type='constant')
</pre><p>Given that this is the default, lets do 'tree'? its more interesting... and also demonstrate how to affect tree depth? (if this happens from here...?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>